packages/safe-ds-eda/types/state.d.ts

```diff
@@ -77,26 +77,25 @@ export interface Profiling {
 
 export interface ProfilingDetailBase {
```

Comment:
Does this need to be exported? It seems like the subtypes + `ProfilingDetail` should be enough.

Reply:
Already addressing this in a new branch where I work out more details of the state and history